### PR TITLE
support closeOnCompletion

### PR DESCRIPTION
**Pull request by kares**
+ missed unwrapping test for previous PR

### CODE REVIEW

File: src/main/java/org/sqlite/jdbc4/JDBC4Statement.java [hunk 1]
```diff
@@ -20,9 +20,28 @@ public boolean isWrapperFor(Class<?> iface) {
...
+        closed = true; // isClosed() should only return true when close() happened
+    }
+
+    public boolean isClosed() {
+        return closed;
+    }
+
+    boolean closeOnCompletion;
+
+    public void closeOnCompletion() throws SQLException {
```
Comment (xerial) [inline src/main/java/org/sqlite/jdbc4/JDBC4Statement.java]: Please add `@Override` to closeOnCompletion() and isCloseOnCompletion()

**Comment by xerial** [comment]
Calling closeOnCompletion etc. breaks jdk6 compatibility. We may need to use reflection for calling these methods.

**Comment by xerial** [comment]
Fixed the test code so that it can work in JDK6. 
https://github.com/xerial/sqlite-jdbc/commit/3123878737d095cbf0478f35a9caea53268913b0

Thanks for the contribution!

**Comment by kares** [comment]
right, missed that - thanks!

**Pull request merged**